src/NServiceBus.AcceptanceTests/Core/SelfVerification/ConventionEnforcementTests.cs

```diff
@@ -38,6 +38,19 @@ public void Ensure_all_messages_are_public()
             CollectionAssert.IsEmpty(missingBaseClass, string.Join(",", missingBaseClass));
         }
 
+        [Test]
+        public void Ensure_all_diagnostics_tests_are_run_sequentially()
+        {
+            var testTypes = Assembly.GetExecutingAssembly().GetTypes().Where(HasTestMethod);
+
+            var diagnosticTests = testTypes
+                .Where(t => t.Namespace == "NServiceBus.AcceptanceTests.Diagnostics")
```

Comment:
so we seem to have some conflicts with the already existing "Diagnostics" namespace. Should we use "OpenTelemetry" instead?